frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -182,11 +182,19 @@ export interface DetailsGridProps
   name: string;
   namespace?: string;
   sectionsFunc?: (item: KubeObject) => React.ReactNode;
+  actions?: JSX.Element[] | ((item: any) => any);
```

Comment:
This is not needed because this `DetailsGridProps` already extends `MainInfoSectionProps` which supports the `actions` property.

Reply:
updated